apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -48,24 +48,24 @@ export class IENApplicantUtilService {
    * @returns retrun promise of find()
    */
   async applicantFilterQueryBuilder(filter: IENApplicantFilterAPIDTO) {
-    const { status, ha_pcn, name } = filter;
+    const { ha_pcn, name } = filter;
     const query: any = {
       order: {
         updated_date: 'DESC',
       },
       relations: this.applicantRelations.status,
     };
-    if (!status && !ha_pcn && !name) {
+    if (!ha_pcn && !name) {
       return this.ienapplicantRepository.find(query);
     }
 
     let where: any = {};
     let isWhere = false;
-    if (status) {
-      const status_list = await this.fetchChildStatusList(status);
-      where = { status: In(status_list), ...where };
-      isWhere = true;
-    }
+    // if (status) {
+    //   const status_list = await this.fetchChildStatusList(status);
+    //   where = { status: In(status_list), ...where };
+    //   isWhere = true;
+    // }
```

Comment:
Commented out code here

Reply:
It's temporary, I'll enable it in the next couple of days, once the milestone/status import completes successfully.